src/chatkit.ts

```diff
@@ -219,7 +220,7 @@ export default class Chatkit {
 
     const sdkInfo = new SDKInfo({
       productName: 'chatkit',
-      version: '0.13.0', // TODO extract from package.json
+      version: packageJSON.version,
```

Comment:
@vivangkumar that was an enormous pain but after much trial and error this works 😅